package.json

```diff
@@ -123,15 +123,15 @@
   "dependencies": {
     "algoliasearch": "3.24.9",
     "algoliasearch-helper": "2.24.0",
-    "classnames": "^2.2.5",
-    "events": "^1.1.0",
-    "hogan.js": "^3.0.2",
-    "lodash": "^4.17.4",
-    "preact": "^8.1.0",
-    "preact-compat": "^3.16.0",
-    "preact-rheostat": "^2.1.1",
-    "prop-types": "^15.5.10",
-    "to-factory": "^1.0.0"
+    "classnames": "2.2.5",
+    "events": "1.1.0",
```

Comment:
This means that as long as we don't do a release, people will be stuck on the older version of these, while with the ^, they'd still get upgraded, even if we don't update yet. I wonder if that's a problem 🤔